db-migrations/37_update_protocol_default.up.sql

```diff
@@ -0,0 +1,12 @@
+-- Updating default protocol for version endpoints
+update 
+    version_endpoints
+set protocol = 'HTTP_JSON'
+where protocol is null or protocol = '';
+
+
+-- Updating default protocol for model endpoints
+update 
+    model_endpoints
+set protocol = 'HTTP_JSON'
+where protocol is null or protocol = '';
```

Comment:
Super tiny nit but can we have the SQL commands in capital letters for consistency with other migration scripts (and for very slightly better readability) 😅 Thanks! 🙏